### DISCUSSION TITLE
Install pyodbc for Postgresql in Ubuntu

### DISCUSSION THREAD

**Discussion by TimdBui** [mkleehammer/pyodbc#1068] [Q&A] [answered]
Hello, I am a new student of Postgres, Python and Ubuntu. Could you please tell me the instruction how to install pyodbc in Ubuntu so my Python can send and receive data from my Postgresql database? Thank you in advance for your instruction!

**Comment by v-chojas** [comment, marked as answer]
[The documentation is here](https://github.com/mkleehammer/pyodbc/wiki)

**Reply by TimdBui** [reply]
Thank you so much for taking the time to help!
